docs/advanced_topics/api/v2/usage.md

```diff
@@ -0,0 +1,647 @@
+                "locale": "en-gb"
+            },
+            "title": "British page"
+        },
+        {
+            "id": 12,
+            "meta": {
+                "type": "standard.StandardPage",
+                "detail_url": "http://api.example.com/api/v2/pages/12/",
+                "html_url": "http://www.example.com/fr-page/",
+                "slug": "fr-page",
+                "first_published_at": "2016-08-30T16:52:00Z",
+                "locale": "fr"
+            },
+            "title": "French page"
+        },
+    ]
+}
+```
+
+### Fields
+
+By default, only a subset of the available fields are returned in the response.
+The `?fields` parameter can be used to both add additional fields to the
+response and remove default fields that you know you won't need.
+
+#### Additional fields
+
+
+Additional fields can be added to the response by setting `?fields` to a
+comma-separated list of field names you want to add.
+
+For example, `?fields=body,feed_image` will add the `body` and `feed_image`
+fields to the response.
+
+This can also be used across relationships. For example,
+`?fields=body,feed_image(width,height)` will nest the `width` and `height`
+of the image in the response.
+
+#### All fields
+^^^^^^^^^^
+
+Setting `?fields` to an asterisk (`*`) will add all available fields to the
+response. This is useful for discovering what fields have been exported.
+
+For example: `?fields=*`
+
+#### Removing fields
+
+Fields you know that you do not need can be removed by prefixing the name with a
+`-` and adding it to `?fields`.
+
+For example, `?fields=-title,body` will remove `title` and add `body`.
+
+This can also be used with the asterisk. For example, `?fields=*,-body`
+adds all fields except for `body`.
+
+#### Removing all default fields
+
+
+To specify exactly the fields you need, you can set the first item in fields to
+an underscore (`_`) which removes all default fields.
+
+For example, `?fields=_,title` will only return the title field.
+
+### Detail views
+
+You can retrieve a single object from the API by appending its id to the end of
+the URL. For example:
+
+-   Pages `/api/v2/pages/1/`
+-   Images `/api/v2/images/1/`
+-   Documents `/api/v2/documents/1/`
+
+All exported fields will be returned in the response by default. You can use the
+`?fields` parameter to customize which fields are shown.
+
+For example: `/api/v2/pages/1/?fields=_,title,body` will return just the
+`title` and `body` of the page with the id of 1.
+
+### Finding pages by HTML path
+
+You can find an individual page by its HTML path using the `/api/v2/pages/find/?html_path=<path>` view.
+
+This will return either a `302` redirect response to that page's detail view, or a `404` not found response.
+
+For example: `/api/v2/pages/find/?html_path=/` always redirects to the homepage of the site
+
+## Default endpoint fields
+
+### Common fields
+
+These fields are returned by every endpoint.
+
+```
+.. glossary::
```

Comment:
[The [documentation ](https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html#glossary) mentions this is a format to avoid. Can you use a plain list?

Reply:
Changes made to match the suggestion